steps/plan/outputs.schema.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "properties": {
+    "plan": {
+      "type": "string",
+      "description": "Object containting a base64-encoded Terraform .tfplan",
```

Comment:
I'd probably just make this something opaque like "Terraform plan data to be used to apply this specific plan" -- people shouldn't really have to worry about the type/data format here.